backend/app/tests/utils/utils.py

```
@@ -9,7 +10,7 @@
 
 from app.core.config import settings
 from app.crud.user import get_user_by_email
-from app.models import APIKeyPublic
+from app.models import APIKeyPublic, Credential
```

Ruff (0.11.9)

`app.models.Credential` imported but unused

Remove unused import: `app.models.Credential`

(F401)